.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Check Advisories
+
+on:
+  # Triggers the workflow on push events only for the default branch
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
The repo main branch is not called 'main' but 'master', are they aliased in GH?

Reply:
Oh right! Good catch. Copy-pasta error. I'll fix.